code/game/objects/items/devices/flashlight.dm

```diff
@@ -201,6 +201,31 @@
 	/// How much sanitization to apply to the burn wound
 	var/uv_power = 1
 
+/obj/item/flashlight/pen/paramedic/advanced
+	name = "advanced penlight"
+	desc = "A stronger version of the UV penlight that paramedics and doctors recieve, it is capable of cauterizing bleeding as well as sterilizing burns."
+	icon_state = "penlight_cmo"
+	light_range = 4
+	uv_power = 2
+	toolspeed = 0.5
+	tool_behaviour = TOOL_CAUTERY
+
+/obj/item/flashlight/pen/paramedic/advanced/ignition_effect(atom/A, mob/user)
+	. = ..()
+	return "[user] holds [src] against [A] until it ignites."
+
+/obj/item/flashlight/pen/paramedic/advanced/afterattack(atom/target, mob/user, proximity_flag)
+	. = ..()
+	if(!proximity_flag)
+		if(holo_cooldown > world.time)
+			to_chat(user, span_warning("[src] is not ready yet!"))
+			return
+		var/T = get_turf(target)
+		if(locate(/mob/living) in T)
+			new /obj/effect/temp_visual/medical_holosign(T,user) //produce a holographic glow
+			holo_cooldown = world.time + 10 SECONDS
+			return
+
```

Comment:
what's the point of this? regular penlights already have this afterattack

Reply:
Fogor that it applied to all penlights and threw it in, didn't realize it'd cause a redundancy